alf/config_helpers.py

```diff
@@ -192,7 +192,7 @@ def adjust_config_by_multi_process_divider(ddp_rank: int,
         config1('TrainerConfig.evaluate', False, raise_if_used=False)
 
 
-def parse_config(conf_file, conf_params):
+def parse_config(conf_file, conf_params, set_random_seed=True):
```

Comment:
It seems to me create_env is better than set_random_seed.

Reply:
Changed argname to create_env